src/index.js

```diff
@@ -26,8 +26,8 @@ const mapRequestToAsset = request => {
 }
 
 const defaultCacheControl = {
-  browserTTL: 0,
-  edgeTTL: 100 * 60 * 60 * 24, // 100 days
+  browserTTL: null,
+  edgeTTL: 2 * 60 * 60 * 24, // 2 days
```

Comment:
lets keep this at 100?

Reply:
it's weird IMO because it would never stay on cache for that long and puts us and developers at risks for situations like this.